packages/@ourworldindata/grapher/src/mapCharts/MapChart.tsx

```diff
@@ -933,15 +1067,16 @@ class ChoroplethMap extends React.Component<{ manager: ChoroplethMapManager }> {
                                 return (
                                     <path
                                         key={feature.id}
-                                        d={feature.path}
+                                        d={this.getPath(feature)}
                                         strokeWidth={
                                             (isFocus ? focusStrokeWidth : 0.3) /
                                             viewportScale
                                         }
                                         stroke={stroke}
                                         strokeOpacity={strokeOpacity}
                                         cursor="pointer"
-                                        fill={`url(#${Patterns.noDataPatternForMapChart}-${this.manager.projection})`}
+                                        // Hashing looks weird as it doesn't rotate with the rest of the globe
```

Comment:
I think this looks clearer than the hashing lines, but could see the arguments for both ways.

Reply:
I think the hashing lines are the right approach here. Any solid colour (including the gray that you have chosen) could be used to indicate a data value, in which case countries with that data value and countries without data would look the same.